File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -4666,13 +4119,12 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Deferred signing without PKCS7 encoding
 
-The following code sample shows deferred signing in a PDF document without PKCS7 encoding from an external signature.
+The following code sample shows deferred signing in a PDF document without PKCS7 encoding from an external signature using [AddExternalSigner](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_AddExternalSigner_Syncfusion_Pdf_Security_IPdfExternalSigner_System_Collections_Generic_List_System_Security_Cryptography_X509Certificates_X509Certificate2__System_Byte___) method.
```

Comment:
The following code sample shows deferred signing in a PDF document without PKCS7 encoding from an external signature using [IPdfExternalSigner]() class and [AddExternalSigner](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_AddExternalSigner_Syncfusion_Pdf_Security_IPdfExternalSigner_System_Collections_Generic_List_System_Security_Cryptography_X509Certificates_X509Certificate2__System_Byte___) method.

Reply:
Changed the content